src/main/java/com/eva/logic/commands/AddApplicationCommand.java

```diff
@@ -53,6 +64,16 @@ public CommandResult execute(Model model) throws CommandException {
 
         Applicant applicantToUpdate = lastShownList.get(index.getZeroBased());
```

Comment:
For this if an invalid index is entered is the error is caught in mainWindow? If it is then i'll change view command to take out the try catch cause it'll make the code look neater

Reply:
The error is caught in lines 61 to 63, in this method itself. If thats the error youre looking for.